spec/controllers/alaveteli_pro/batch_downloads_controller_spec.rb

```diff
@@ -90,9 +90,15 @@ def show(id: '1', format: 'abc')
           end
 
           it 'returns content disposition' do
-            expect(response.header['Content-Disposition']).to(
-              eq 'attachment; filename="NAME"'
-            )
+            if rails_upgrade?
+              expect(response.header['Content-Disposition']).to(
+                eq 'attachment; filename="NAME"; filename*=UTF-8\'\'NAME'
```

Comment:
Are the backslashes here escaping the single quotes?

Might be clear to put this inside a `%q()` string